test/edu-benefits/1990/01-all-fields.e2e.spec.js

```diff
@@ -2,7 +2,7 @@ const E2eHelpers = require('../../e2e/e2e-helpers');
 const Timeouts = require('../../e2e/timeouts.js');
 const EduHelpers = require('../../e2e/edu-1990-helpers');
 
-module.exports = E2eHelpers.createE2eTest(
```

Comment:
No longer exporting this because it's no longer necessary?

Reply:
Yeah, I don't want to run because it expect the old 1990 at the same url. It'll be removed completely later.